chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/urlStatus/data/UrlResult.java

```diff
@@ -45,47 +51,12 @@ public boolean equals(Object obj) {
         }
         return false;
     }
+
     @Override
     public int hashCode() {
         if (StringUtils.isEmpty(this.getUrl()) || this.getLastChecked() == null) {
             return -1;
         }
         return this.getUrl().hashCode() + this.getLastChecked().hashCode();
     }
```

Comment:
The `@Data` lombok annotation generates a `hashCode()` method.  Could we just use that instead of this one, or is there something special about this method?

Reply:
I don't know.. I didn't change any functionality in this class other than moving it into a different package to better match the structure we follow and bring it up-to-date with our lombok Data, NoArgsConstructor, etc usage. This class is used in the "data collection" job which fills in the db with data that the Emailer job uses and the ticket was only to change the behavior of the Emailer job. I tried to bring all these related classes more in line with our coding standards without changing their behavior and I would rather not remove these equals and hashCode implementations since they do something other than the default behavior of comparing every field that Lombok would have them do. I think if we need/want to get rid of these methods then we'd need to regression test the data collection job as well.